templates/cosmic-microservices.yaml

```diff
@@ -1,4 +1,8 @@
 {{- $context := . -}}
+{{- range .Values.namespaces -}}
```

Comment:
Indeed found out was missing a file, will add it.
Will have another look at the namespace